index.html

```diff
@@ -2521,10 +2521,10 @@ <h3>Device Fingerprinting</h3>
 
         <p>
 There are mechanisms external to Verifiable Credentials that are used to track and
-correlate individuals on the internet and the Web. Some of these mechanisms
-include Internet Protocol address tracking, web browser fingerprinting,
-evercookies, advertising network trackers, mobile network position information,
-and in-application global positioning system APIs. The use of Verifiable
+correlate individuals on the Internet and the Web. Some of these mechanisms
+include Internet Protocol address tracking, Web Browser fingerprinting,
```

Comment:
Oops, I just intended for you to only change Internet and GPS back... Web Browser, Evercookies, and Advertising Network should have all been lower case. I'm fine w/ just pulling it in and fixing later if you don't want to do another commit.